### PR TITLE
AP-1775: webpack for jquery

### PR DESCRIPTION
**Pull request by colinbruce**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-1775)

Removed the static jquery folder from public
Installed and configured webpack
Added jquery using yarn
Updated rubocop to ignore node_modules

## Checklist

Before you ask people to review this PR:

- [x] Tests and rubocop should be passing: `bundle exec rake`
- [x] Github should not be reporting conflicts; you should have recently run `git rebase master`.
- [x] There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- [x] The PR description should say what you changed and why, with a link to the JIRA story.
- [x] You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- [x] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: app/views/layouts/apipie_override.html.erb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  <div class="row-fluid">
+    <div id='container'>
+      <%= yield %>
+      <%= render 'disqus' if Apipie.configuration.use_disqus? %>
+    </div>
+  </div>
+  <hr>
+  <footer><%= yield :apipie_footer %></footer>
+</div>
+<script type='text/javascript' src='/apidocs/javascripts/bundled/jquery.js'></script>
```
Comment (Obsiye) [inline app/views/layouts/apipie_override.html.erb:25]: Do we need this line since jquery is in yarn

**Comment by Obsiye** [comment]
I tested this on UAT and in the broswer console it came up with errors. I think because config/initializers/secure_headers.rb isn't allowing apidocs to call it's public files so it needs updating.

File: app/views/layouts/apipie_override.html.erb (hunk 1 above)
Comment (colinbruce) [reply app/views/layouts/apipie_override.html.erb:25]: Good catch, I managed to replace the wrong line! 🤦

File: app/views/layouts/apipie_override.html.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  <div class="row-fluid">
+    <div id='container'>
+      <%= yield %>
+      <%= render 'disqus' if Apipie.configuration.use_disqus? %>
+    </div>
+  </div>
+  <hr>
+  <footer><%= yield :apipie_footer %></footer>
+</div>
+<%= javascript_pack_tag 'application' %>
```
Comment (Obsiye) [inline app/views/layouts/apipie_override.html.erb:25]: so clever. great way of getting around the issue.

**Review by Obsiye — APPROVED** [review]
Awesome fix!

**Pull request merged**